### ISSUE TITLE
Iteration Zero fails on bare-metal OCP cluster

### ISSUE THREAD

**Issue by binnes** [cloud-native-toolkit/ibm-garage-iteration-zero#373]
When doing a fast-install on a bare metal OCP cluster (4.9) - no cloud provider, I get the following error:

```
│ Error: Missing required argument
│ 
│   on stage2-image-registry.tf line 1, in module "dev_tools_ibm_image_registry":
│    1: module "dev_tools_ibm_image_registry" {
│ 
│ The argument "region" is required, but no definition was found.
╵
╷
│ Error: Unsupported argument
│ 
│   on stage2-image-registry.tf line 5, in module "dev_tools_ibm_image_registry":
│    5:   cluster_region      = module.dev_cluster.region
│ 
│ An argument named "cluster_region" is not expected here.
```

**Version**
used command `curl -sfL get.cloudnativetoolkit.dev | sh -` to do the install with latest iteration-zero release 2.18.0

I didn't make any modifications, so all default values being used (this command ran 2 weeks ago without any issues)

**Comment by CayoM** [comment]
im getting the same issue. The problem here is that the ressource got renamed here:

https://github.com/cloud-native-toolkit/terraform-ibm-image-registry/blob/main/test/stages/stage2-image-registry.tf

I just created a pull request on the source to use the old naming "cluster_region" instead of the new "region" which introduced this breaking change.

https://github.com/cloud-native-toolkit/terraform-ibm-image-registry/pull/27

**Comment by boldhawk** [comment]
Any update on this? Its currently blocking a number of engagements. How did this get through any testing we did before being merged to the master branch?

**Comment by binnes** [comment]
fixed in #380
